src/views/conference/2018/index/l10n.json

```diff
@@ -11,6 +11,7 @@
 
     "conference-2018.desc1": "Join us for the Scratch@MIT conference, a playful gathering of educators, researchers, developers, and other members of the worldwide Scratch community.",
     "conference-2018.desc2": "We're planning a very participatory conference, with an entire day of hands-on workshops and lots of opportunities for peer-to-peer discussion and collaboration.",
+    "conference-2018.desc3": "The conference is intended primarily for adults who support young people learning Scratch."
```

Comment:
Can this not just be added to the end of the line above?

Reply:
You're right, I do that.
Can It Be Added Though?